recipes/libgd/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "2.2.1" %}
+
+package:
+  name: libgd
+  version: {{ version }}
+
+source:
+  fn: libgd-{{ version }}.tar.gz
+  url: https://github.com/libgd/libgd/releases/download/gd-{{version}}/libgd-{{ version }}.tar.gz
+  sha256: a763df35d653683beb79b9bca1944652b32abace9c8a5a93d5439841f3c1da89
+  patches:
+    - osx_version.patch
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - pkg-config
+    - automake
+    - gettext
+    - zlib 1.2*
+    - freetype 2.6*
+    - jpeg 9*
+    - libpng 1.6*
+    - libtiff 4.0.*
+    - fontconfig 2.11.*
+    - libiconv
+    - expat
+  run:
+    - zlib 1.2*
+    - freetype 2.6*
+    - jpeg 9*
+    - libpng 1.6*
+    - libtiff 4.0.*
+    - fontconfig 2.11.*
+    - libiconv
+    - expat
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libgd.a      # [linux]
+    - test -f $PREFIX/lib/libgd.dylib  # [osx]
+    - test -f $PREFIX/lib/libgd.la
+about:
+    home: http://libgd.github.io/
+    summary: library for the dynamic creation of images
+    license: Custom OSS
```

Comment:
According to [this](https://github.com/libgd/libgd/issues/47) it should be `GD license`. You can also add a `license_file` metadata pointing to the file in the source dir. See [zlib](https://github.com/conda-forge/zlib-feedstock/blob/master/recipe/meta.yaml) for an example.